packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1260,4 +1260,18 @@
       <protected>false</protected>
       <shortDescription>Something went wrong and we couldn't generate an answer.</shortDescription>
     </labels>
+    <labels>
+      <fullName>quantic_ShowingResultsFor</fullName>
+      <value>Showing results for {{0}}</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Showing results for example</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_SearchInsteadFor</fullName>
+    <value>Search instead for</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Search instead for</shortDescription>
+  </labels>
```

Comment:
Why did we need this now?

Reply:
These are specific labels that we use when the Query is updated by a pipeline trigger, when instead the query is corrected because it contained a typo or something, other labels will be used/. for example: `Query was automatically corrected to "example correct query"